src/main/java/uk/gov/hmcts/reform/authorisation/ServiceAuthAutoConfiguration.java

```diff
@@ -16,4 +22,13 @@ public ServiceAuthHealthIndicator serviceAuthHealthIndicator(ServiceAuthorisatio
         return new ServiceAuthHealthIndicator(coreCaseDataApi);
     }
 
+    @Bean
+    @ConditionalOnProperty("idam.s2s-authorised.services")
```

Comment:
I would put this under: 

‘s2s.authorised_services’

Could make a breaking change for the s2s url to be consistent, leave it as is, or maybe there’s a way to alias it